figures/S3.supplemental_figure3/scripts/S3.r

```diff
@@ -0,0 +1,1060 @@
+    ))
+
+
+# create grouping of treatment and group
+cytokine_values$group_treatment <- paste(cytokine_values$oneb_Treatment_Dose_Inhibitor_Dose, cytokine_values$group,  sep = ", ")
+# make the group_treatment column a factor
+cytokine_values$group_treatment <- factor(
+    cytokine_values$group_treatment,
+    levels = c(
+        'DMSO 0.1%, Control',
+        'Flagellin 0.1 ug/ml, Control',
+        'Flagellin 1.0 ug/ml, Pyroptosis',
+        'LPS 0.01 ug/ml, Pyroptosis',
+        'LPS 0.1 ug/ml, Pyroptosis',
+        'LPS 1.0 ug/ml, Pyroptosis',
+        'LPS 10.0 ug/ml, Pyroptosis',
+        'LPS 100.0 ug/ml, Pyroptosis',
+        'LPS 1.0 ug/ml + Nigericin 1.0 uM, Pyroptosis',
+        'LPS 1.0 ug/ml + Nigericin 3.0 uM, Pyroptosis',
+        'LPS 1.0 ug/ml + Nigericin 10.0 uM, Pyroptosis',
+        'H2O2 100.0 nM, Control',
+        'H2O2 100.0 uM, Control',
+        'Thapsigargin 1.0 uM, Apoptosis',
+        'Thapsigargin 10.0 uM, Apoptosis'
+    )
+)
+unique(cytokine_values$group_treatment)
+
+# set plot size
+width <- 10
+height <- 5
+options(repr.plot.width=width, repr.plot.height=height)
+cytokine_scatter_plot1 <- (
+    ggplot(
+        data = cytokine_values,
+        aes(
+            x = `IL-1 beta [NSU]`,
+            y = `CCL24 [NSU]`,
+            color = group_treatment,
+            shape = group_treatment
+        )
+    )
+    + geom_point(size = 5, alpha = 0.7)
+    + theme_bw()
+    + xlab("IL-1 beta Abundance")
+    + ylab("CCL4 alpha Abundance")
+    + xlim(-0.1, 1)
+    + ylim(-0.1, 1)
+    # rename legend title
+    + labs(color = "Treatment", hjust = 0.5)
+    + figure_theme
+        + theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust=1))
+            + theme(
+         legend.text = element_text(size = 16),
+        legend.title = element_text(size = 20, hjust = 0.5))
+    + scale_color_manual(
+        name = "Treatment",
+        labels = c(
+        'DMSO 0.1%, Control',
+        'Flagellin 0.1 ug/ml, Control',
+        'Flagellin 1.0 ug/ml, Pyroptosis',
+        'LPS 0.01 ug/ml, Pyroptosis',
+        'LPS 0.1 ug/ml, Pyroptosis',
+        'LPS 1.0 ug/ml, Pyroptosis',
+        'LPS 10.0 ug/ml, Pyroptosis',
+        'LPS 100.0 ug/ml, Pyroptosis',
+        'LPS 1.0 ug/ml + Nigericin 1.0 uM, Pyroptosis',
+        'LPS 1.0 ug/ml + Nigericin 3.0 uM, Pyroptosis',
+        'LPS 1.0 ug/ml + Nigericin 10.0 uM, Pyroptosis',
+        'H2O2 100.0 nM, Control',
+        'H2O2 100.0 uM, Control',
+        'Thapsigargin 1.0 uM, Apoptosis',
+        'Thapsigargin 10.0 uM, Apoptosis'
+        ),
+        values = c(
+        '#66c2a5',
+        '#3288bd',
+        '#f46d43',
+        '#5e4fa2',
+        "#058ED9",
+        "#848FA2",
+        "#2D3142",
+        "#FFC857",
+        '#66c2a5',
+        '#3288bd',
+        '#abdda4',
+        '#fdae61',
+        # dark grey
+        '#666666',
+        # green
+        '#66c2a5',
+        '#f46d43'
+        # '#d53e4f'
```

Comment:
Recommend removing commented out code if not needed.

Reply:
removed